conf/variant/intel-qtauto/bblayers.conf.sample

```diff
@@ -0,0 +1,13 @@
+# These are needed for bitbake to pick up the includes
+BSPDIR := "${TOPDIR}/../"
```

Comment:
The last "/" is not needed. It's fine by me, though, unless you have a policy against it.

Reply:
Good point, will remove it.